packages/histoire-controls/package.json

```diff
@@ -34,6 +34,7 @@
     "build": "rimraf dist && pnpm run build:css && vite build && pnpm run build:types",
     "build:css": "tailwindcss -i src/style/main.css -o ./dist/style-standalone.css -c tailwind.config.cjs --postcss postcss.config.cjs",
     "build:types": "vue-tsc --declaration --emitDeclarationOnly",
+    "story:dev": "histoire dev",
```

Comment:
Revert this file

Reply:
Done here: https://github.com/histoire-dev/histoire/pull/310/commits/67c61e29a3b9e75c71f4a06c2ae1e29712259a99